lib/kitchen/directions/bake_non_introduction_pages.rb

```diff
@@ -3,16 +3,25 @@
 module Kitchen
   module Directions
     module BakeNonIntroductionPages
-      def self.v1(chapter:, add_target_label: false)
+      def self.v1(chapter:, custom_target_label: false, cases: false)
         chapter.non_introduction_pages.each do |page|
           number = "#{chapter.count_in(:book)}.#{page.count_in(:chapter)}"
 
           page.search("div[data-type='description']").each(&:trash)
           page.add_class('chapter-content-module')
 
-          page.target_label(custom_content: number) if add_target_label
-
           title = page.title
+          title_label = title.children
+
+          if custom_target_label
+            page.custom_target_label_for_modules(custom_title_content: " #{title_label}",
+                                                 custom_number_content: number)
+          elsif cases
+            page.target_label(label_text: 'module', custom_content: "#{number} #{title_label}", cases: cases)
+          else
+            page.target_label(custom_content: "#{number} #{title_label}")
+          end
+
```

Comment:
I have a few questions here. 
1. Before target label was optional. With current condition there will be always defined. Is it intentional?
2. `target_label` method already has `else if` condition in its definition, so it shouldn't be called twice here. I understand that problem is with `label_text`, because it appears only in Polish books. I'm wondering how we can fix method definition to cover such cases :thinking: Or maybe using `label_text: {'module' if cases}` here could help?